### PR TITLE
add basic README content

### PR DESCRIPTION
**Pull request by micbar**
# Description

add basic README.md content.

- Contributing
- Writing new tests
- Usage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,151 @@
...
+
+## End-to-End Test Suite for the CS3 APIs
+
+**This tool will receive a lot of changes before version 1.0.0**
+
+The cs3api-validator is a tool to test implementations of the [CS3Apis](https://github.com/cs3org/cs3apis). It works as a standalone software which only needs the address of a running cs3api provider.
```
Comment (phil-davis) [inline README.md]:
```suggestion
The cs3api-validator is a tool to test implementations of the [CS3Apis](https://github.com/cs3org/cs3apis). It works as standalone software which only needs the address of a running cs3api provider.
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,151 @@
...
+Here are some useful channels to try:
+
+- [#ocis](https://talk.owncloud.com/channel/ocis) - Next Generation ownCloud Architecture
+- [#general](https://talk.owncloud.com/channel/general) - General Topics around File Sync & Share
+
+## Developing
+
+### Quick start
+```shell
+git clone git@github.com:owncloud/cs3api-validator.git
```
Comment (phil-davis) [inline README.md:45]:
```
$ git clone git@github.com:owncloud/cs3api-validator.git
Cloning into 'cs3api-validator'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
$ git clone https://github.com/owncloud/cs3api-validator.git
Cloning into 'cs3api-validator'...
remote: Enumerating objects: 163, done.
remote: Counting objects: 100% (163/163), done.
remote: Compressing objects: 100% (113/113), done.
remote: Total 163 (delta 79), reused 116 (delta 44), pack-reused 0
Receiving objects: 100% (163/163), 97.49 KiB | 486.00 KiB/s, done.
Resolving deltas: 100% (79/79), done.
```

What am I doing wrong. The suggested command gives me errors, but I can clone https://github.com/owncloud/cs3api-validator.git

File: README.md [hunk 3]
```diff
@@ -1,4 +1,151 @@
...
+
+#### Build a binary with the tests
+
+Run ``go test -c``. This will create a ``cs3api-validator.test`` binary.
+
+Execute the tests ``./cs3qpi-validator.test --endpoint=your-addr:port``
+
+### Use tags
+
+You can use [tags](https://github.com/cucumber/godog#tags) to filter features which should be exectuted. `--godog.tags=<expression>`
```
Comment (phil-davis) [inline README.md]:
```suggestion
You can use [tags](https://github.com/cucumber/godog#tags) to filter features which should be executed. `--godog.tags=<expression>`
```

File: README.md (hunk 2 above)
Comment (micbar) [reply README.md:45]: seems that http works and ssh not.

**Review by wkloucek — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]

**Pull request merged**